### ISSUE TITLE
Break trigger event

### ISSUE THREAD

**Issue by TMDevel89** [MIvanchev/NppEventExec#5]
Is there a possibility in the script to cancel the triggered event?

For example: Event BeforeSave is triggered, Script A is called, Script A checks condition, condition is false > Script A aborts the save process

**Comment by MIvanchev** [comment]
No, this is also controlled by Notepad++ and I believe well outside the scope of the plugin API. It could be emulated by saving the previous content and restoring it if the script decides so.
